index.html

```
@@ -724,10 +814,11 @@ <h3 class="text-2xl font-bold text-zinc-900 dark:text-white mb-2">No organizatio
   </div>
 
   <div class="mt-12 flex justify-center" id="loadMoreContainer" style="display:none">
```

HTMLHint (1.9.2)

[error] The id value [ loadMoreContainer ] must be unique.

(id-unique)